packages/design-system-react-native/src/components/BadgeNetwork/README.md

```diff
@@ -0,0 +1,163 @@
+# BadgeNetwork
+
+The `BadgeNetwork` component is reserved for badged representing networks. It provides a fallback mechanism in case of an image load failure, ensuring a graceful degradation by displaying either an alternative text or a fallback icon.
+
+---
+
+## Props
+
+The `BadgeNetwork` component accepts the following props:
+
+### `src` (Required)
+
+The source of the image or SVG. It determines whether a local image, a local SVG component, or a remote image/SVG (via URI) is rendered.
+
+| TYPE                                                    | REQUIRED | DEFAULT |
+| :------------------------------------------------------ | :------- | :------ |
+| `number \| ComponentType<SvgProps> \| { uri?: string }` | Yes      | `N/A`   |
+
+---
+
+### `name` (Optional)
+
+Used to generate fallback text when the image or SVG fails to load.
+
+| TYPE     | REQUIRED | DEFAULT     |
+| :------- | :------- | :---------- |
+| `string` | No       | `undefined` |
```

Comment:
If name is used for the fallback if the image fails should it be required?

Reply:
it's currently not required in mobile